src/report.py

```diff
@@ -0,0 +1,98 @@
+import csv
+from typing import List, Dict, Tuple
+
+
+def read_portfolio(filename: str) -> List[Dict]:
+    """
+    Reads a portfolio file and places each entry into a list.
+
+    :param filename: Name of the portfolio file
+    :type filename: str
+    :raises ValueError: Raises ValueError if line in the file is empty
+    :return: Returns list of portfolio entries
+    :rtype: List[Dict]
+    """
+    portfolio = []
+    with open(filename) as f:
+        rows = csv.reader(f)
+        headers = next(rows)
+        for row in rows:
+            if not row:
+                raise ValueError
+            newrow = (row[0], int(row[1]), float(row[2]))
+            portfolio.append(dict(zip(headers, newrow)))
+
+    return portfolio
+
+
+def read_prices(filename: str) -> Dict:
+    """
+    Reads a prices file and places each entry into a dictionary.
+
+    :param filename: Name of the prices file
+    :type filename: str
+    :return: Returns a dictionary of prices and their stock names
+    :rtype: Dict
+    """
+    prices = {}
+    with open(filename) as f:
+        rows = csv.reader(f)
+        for row in rows:
```

Comment:
in the same way as suggested in `read_portfolio` you can do the following:
```suggestion
        for stock_name, price in rows:
```